ui/src/app/utils/strings.ts

```diff
@@ -0,0 +1 @@
+export const ACTION_DISABLED_INVALID_BILLING = 'A valid billing account is required for this action';
```

Comment:
I can see this being controversial but I felt like we don't really have a good way to organize shared code on the front end. Ex. no `BillingService` to shove this into.

I've used this kind of approach with a global string file when I worked with Android and felt like it worked pretty well.

Reply:
I'm ok with this, I've seen it work before when the number of strings was pretty small. If you end up with more than ten lines in here you want to start thinking about why the string can't be part of a tiny component or whatever but this is totally fine.